### PR TITLE
DOCS-11264 - delivery reliability

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -14,14 +13,19 @@ Using Anypoint MQ, you can create two types of queues:
 These queues don't guarantee a specific message order. Standard queues are the best fit for applications in which messages must be delivered quickly.
 +
 Anypoint MQ supports up to 120,000 in-flight messages per standard queue.
-
++
+Standard queues provide "at least once" message-delivery reliability.
```
Comment (schinthakindims) [inline modules/ROOT/pages/mq-queues.adoc:17]: @hannanelson , should we add that a message can be delivered more than once *occasionally* due to highly distributed architecture?
@prafulrana

Comment (prafulrana) [reply modules/ROOT/pages/mq-queues.adoc:17]: Yeah adding that color will help.

**Review by prafulrana — DISMISSED** [review]

File: modules/ROOT/pages/mq-queues.adoc [hunk 2]
```diff
@@ -14,14 +13,20 @@ Using Anypoint MQ, you can create two types of queues:
...
+Anypoint MQ can deliver each message to more than one consumer.
```
Comment (schinthakindims) [inline modules/ROOT/pages/mq-queues.adoc]: Is this line redundant given next line ?

Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc:18]: I don't think so - it's an "in other words" clarification. If you feel it's unnecessary, I can remove it.

Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc]: OK, removed.

**Review by schinthakindims — APPROVED** [review]
LGTM

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**